twitter/komand_twitter/actions/block/action.py

```diff
@@ -30,7 +31,7 @@ def run(self, params={}):
 
         # Return False if no object was returned - we can't be certain the block was successful.
         if not blocked_user:
-            raise Exception("No object returned from Twitter. Uncertain of block success. ")
+            raise PluginException(cause="No object returned from Twitter. Uncertain of block success.")
```

Comment:
This plugin is using Python 2 SDK. It doesn't support PluginException.

Verify all the plugins pass references with `icon-plugin run -c info`